### DISCUSSION TITLE
Single item per row in Recent Sessions

### DISCUSSION THREAD

**Discussion by pixelstuff** [rustdesk/rustdesk#4731] [Q&A] [answered]
Can we get a single item per row layout, like a spreadsheet view, for for the systems in the Recent Sessions and Favorites tabs?

Currently in v1.9 it seems to only offer a Tile and Compact Tile view that truncates long names and makes scanning the list more convoluted when you have 20+ systems in there.

**Comment by gvaiou** [comment]
I second that, or at least let us change the tile size, or number of tiles per line (provided the tiles get bigger to cover the whole line

**Comment by SergeyMy** [comment]
much less? and so 2 lines
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/fe0c8fa5-839f-4607-a8a6-1d3cb3c108be)

**Comment by pixelstuff** [comment]
I was thinking something along these lines so the long names with similar beginning can be more easily differentiated. Also, scanning a single column is much easier than scanning rows and columns together.

![mockup](https://github.com/rustdesk/rustdesk/assets/428855/35f1a25d-d2e0-4748-964f-0b09a108baea)

**Comment by rustdesk** [comment]
yes, maybe we can provide another mode, list mode, actually anydesk has 3 modes, one is single item per line, @21pages , check it please.

**Reply by rustdesk** [reply]
low  priority, @21pages

**Reply by rustdesk** [reply]
![image](https://github.com/rustdesk/rustdesk/assets/71636191/08608e40-369e-420c-b953-f41010a2b353)

**Comment by rustdesk** [comment]
@sahilyeole

**Comment by Tw33ki** [comment]
We often have more than one maschine on remote clientnetwork that we need to access. When we name them in Rustdesk-Adressbook it looks like that:

![grafik](https://github.com/rustdesk/rustdesk/assets/69896243/3fb7bc99-8fc1-4560-826f-24a29d45edb4)

Unforutnalty there isnt an possibility to see which ist which.

May u be able to fix that one?

**Reply by rustdesk** [reply]
`Single item per row` can fix this. @sahilyeole will work on this new feature.

**Comment by beelchester** [comment]
Is this good enough? Or should I make more changes?

Also, I am planning to make it as a third option named 'List View' and rename the current list view to 'Tile View' with Icons similar to anydesk. Any other ideas, let me know.
<img width="1210" alt="Screenshot 2023-10-18 at 2 28 07 AM" src="https://github.com/rustdesk/rustdesk/assets/73148455/80a9c814-7140-44a0-9f64-a3b3bf0f571f">

**Reply by rustdesk** [reply]
You can copy anydesk here.

> Also, I am planning to make it as a third option named 'List View' and rename the current list view to 'Tile View' with Icons similar to anydesk. Any other ideas, let me know.

**Reply by rustdesk** [reply]
Make sure no overflow (very long text).

**Comment by rustdesk** [comment, marked as answer]
done https://github.com/rustdesk/rustdesk/pull/6165
